service/history/historyEngine2_test.go

```diff
@@ -230,8 +231,8 @@ func (s *engine2Suite) TestRecordWorkflowTaskStartedSuccessStickyExpired() {
 	response, err := s.historyEngine.RecordWorkflowTaskStarted(context.Background(), &request)
 	s.Nil(err)
 	s.NotNil(response)
+	s.True(response.StartedTime.After(*expectedResponse.ScheduledTime))
 	expectedResponse.StartedTime = response.StartedTime
-	expectedResponse.ScheduledTime = &time.Time{}
```

Comment:
ScheduledTime should match event time. This was a bug hidden by missing field in `copyWorfklowExecutionInfo`